packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -127,16 +140,64 @@ export class CustomScrollbar extends Component<Props> {
         // Before these where set to inherit but that caused problems with cut of legends in firefox
         autoHeightMax={autoHeightMax}
         autoHeightMin={autoHeightMin}
-        renderTrackHorizontal={this.renderTrackHorizontal}
-        renderTrackVertical={this.renderTrackVertical}
-        renderThumbHorizontal={this.renderThumbHorizontal}
-        renderThumbVertical={this.renderThumbVertical}
-        renderView={this.renderView}
+        renderTrackHorizontal={(passedProps: any) => this.renderTrackHorizontal(passedProps, styles)}
+        renderTrackVertical={(passedProps: any) => this.renderTrackVertical(passedProps, styles)}
+        renderThumbHorizontal={(passedProps: any) => this.renderThumbHorizontal(passedProps, styles)}
+        renderThumbVertical={(passedProps: any) => this.renderThumbHorizontal(passedProps, styles)}
+        renderView={(passedProps: any) => this.renderView(passedProps, styles)}
       >
         {children}
       </Scrollbars>
     );
   }
 }
 
-export default CustomScrollbar;
+export const CustomScrollbar = withTheme(UnthemedCustomScrollbar);
+CustomScrollbar.displayName = 'CustomScrollbar';
+
+const getStyles = stylesFactory(
+  (theme: GrafanaTheme, isPageScrollbar?: boolean): StylesInterface => {
+    return {
+      customScrollbar: css`
+        display: flex;
+        flex-grow: 1;
+      `,
+      trackVertical: css`
+        border-radius: ${theme.border.radius.md};
+        width: ${theme.spacing.sm} !important;
+        right: ${isPageScrollbar ? 0 : 2}px;
+        bottom: ${theme.spacing.xxs};
+      `,
+      trackHorizontal: css`
+        border-radius: ${theme.border.radius.md};
+        height: ${theme.spacing.sm} !important;
+        right: ${theme.spacing.xxs};
+        bottom: ${theme.spacing.xxs};
+        left: ${theme.spacing.xxs};
+      `,
+      thumbVertical: css`
+        linear-gradient(#404357, ${theme.palette.dark10});
+        border-radius: ${theme.border.radius.md};
+        opacity: 0;
+        &:hover {
+          opacity: 0.8;
+          transition: opacity 0.3s ease-in-out;
+        }
+      `,
+      thumbHorizontal: css`
+        linear-gradient(#404357, ${theme.palette.dark10});
```

Comment:
not valid css, have to set a background property

Reply:
oh...didn't see that coming 😅
Thanks alot Torkel...would fix it right away